pkg/tektonlog/tektonlog.go

```diff
@@ -211,91 +211,79 @@ func (t *TektonLogger) GetRunningBuildLogs(ctx context.Context, pa *v1.PipelineA
 	return ch
 }
 
-type podTime struct {
+type stageTime struct {
 	name      string
```

Comment:
maybe we can rename this to `podName` since it stores the name of the pod? because with the new type name, we can think that it's the stage name

Reply:
Previously it was just information about pod only, but right now it's information about the stage and `podName` is one of the attributes of the `Stage` in my opinion.

Reply:
yes, I was talking about the `name` field, not the `stageTime` type. but it's minor